### PR TITLE
[Suggestion] - Add service account annotation

### PR DESCRIPTION
**Pull request by briannosaurus**
### Suggestion

Please add service account annotation so that it can be used with eks secrets driver to mount the secrets store.

Product
Jira, Confluence, Bitbucket

Code of Conduct

- [x]  I agree to follow this project's Code of Conduct

### CODE REVIEW

File: src/main/charts/bitbucket/values.yaml [hunk 1]
```diff
@@ -76,6 +76,9 @@ serviceAccount:
   #
   imagePullSecrets: []
   # - name: secretName
+
+  #Annotations to add to the ServiceAccount if create is true
```
Comment (nanux) [inline src/main/charts/bitbucket/values.yaml:80]:
```suggestion
  # -- Annotations to add to the ServiceAccount (if created)
```

Please apply to all `values.yaml` files.

**Comment by nanux** [comment]
Thank you @briannosaurus! Could you please apply the change to the rest of the helm charts (crowd, bamboo, bamboo-agent)? The charts should have feature parity if it is possible (like in this case).

Can you also add a test to `src/test/java/test/ServiceAccountTest.java`?

Thank you

**Comment by nanux** [comment]
@briannosaurus, I have modified the PR to move the annotations to the metadata stanza. I also added the functionality to the other products.

Could you please sign the CLA?

**Review by errcode1202 — APPROVED** [review]

**Review by nghazali — APPROVED** [review]

**Comment by briannosaurus** [comment]
I have signed the form, thank you for making the change

**Comment by nghazali** [comment]
@briannosaurus, thanks for the contribution.

**Pull request merged**